### PR TITLE
Added Java Calendar getWeeksInWeekYear() function

### PR DESCRIPTION
**Pull request by ATREAY**
### Description

This pull request pertains to issue #3099(Java Calendar getWeeksInWeekYear())

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+Title: '.getWeeksInWeekYear()'
+Description: 'Returns the total number of weeks in a week year in the integer form.'
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md:3]:
```suggestion
Description: 'Returns the total number of weeks in a week year.'
```

File: content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.getWeeksInWeekYear()`** method of the `Calendar` class returns the integer `number of weeks` in `week year`.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md:14]:
```suggestion
The **`.getWeeksInWeekYear()`** method of the `Calendar` class returns an integer representing the number of weeks in a *week year*.
```

File: content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Syntax
+
+```pseudo
+calendar.getWeeksInWeekYear()
+```
+
+> **Note:** The concept of a `week-based year` is important in some calendar systems, especially when dealing with fiscal years or other non-standard year definitions. The number of weeks in a week-based year can vary depending on factors such as the `starting day` of the week and whether the year contains `partial weeks` at the beginning or end.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md:22]:
```suggestion
> **Note:** The concept of a *week-based year* is important in some calendar systems, especially when dealing with fiscal years or other non-standard year definitions. The number of weeks in a week-based year can vary depending on factors such as the starting day of the week and whether the year contains partial weeks at the beginning or end.
```

File: content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+public class CalendargetWeeksInWeekYearExample {
+  public static void main(String args[]){
+
+    // Creating a calendar object
+    Calendar cal = Calendar.getInstance();
+
+    // Displaying the calendar
+    System.out.println("Current Calendar: " + cal.getTime());
+
+    System.out.println("Weeks in week year :"  + cal.getWeeksInWeekYear());
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md:39]:
```suggestion
    System.out.println("Weeks in week year: "  + cal.getWeeksInWeekYear());
```

File: content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+    cal.setWeekDate(1996, 23, 6 );
+
+    System.out.println("Changed Time:"+cal.getTime());
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md:43]:
```suggestion
    System.out.println("Changed Time: "+cal.getTime());
```

File: content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+    // Get the number of weeks in the week-based year
+    System.out.println("Weeks in week year :"  + cal.getWeeksInWeekYear());
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md:46]:
```suggestion
    System.out.println("Weeks in week year: "  + cal.getWeeksInWeekYear());
```

**Review by caupolicandiaz — APPROVED** [review]
@ATREAY I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by ATREAY** [comment]
Hello @caupolicandiaz, 
I have implemented the necessary modifications. I kindly request that you review them.

**Review by caupolicandiaz — APPROVED** [review]
@ATREAY Looks good, this is ready for a second review.

File: content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+The **`.getWeeksInWeekYear()`** method of the `Calendar` class returns an integer representing the number of weeks in a _week year_.
+
+## Syntax
+
+```pseudo
+calendar.getWeeksInWeekYear()
+```
+
+> **Note:** The concept of a _week-based year_ is important in some calendar systems, especially when dealing with fiscal years or other non-standard year definitions. The number of weeks in a week-based year can vary depending on factors such as the starting day of the week and whether the year contains partial weeks at the beginning or end.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getWeeksInWeekYear/getWeeksInWeekYear.md:22]:
```suggestion
> **Note:** The concept of a **week-based year** is important in some calendar systems, especially when dealing with fiscal years or other non-standard year definitions. The number of weeks in a week-based year can vary depending on factors such as the starting day of the week and whether the year contains partial weeks at the beginning or end.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ATREAY I did a second review and left a few comments.

File: content/java/concepts/calendar/terms/isLenient/isLenient.md
[the remark this answers is not in the document]
Comment (ATREAY) [reply content/java/concepts/calendar/terms/isLenient/isLenient.md:1]: Hey @SSwiniarski, I've deleted the isLenient.md file. Could you take a look at it?

**Review by SSwiniarski — APPROVED** [review]
@ATREAY Looks good for a second review.

**Pull request merged**